cairo/ethereum/utils/numeric.cairo

```diff
@@ -109,6 +110,14 @@ func _taylor_exponential{range_check_ptr}(
     let output = output + numerator_accumulated;
     let value = numerator_accumulated * numerator;
     let div = denominator * i;
+
+    let saturate = is_le_felt(2 ** 128, value);
+    if (saturate != 0) {
+        // Return current accumulated output/denominator when we hit saturation
+        let (res, _) = divmod(output, denominator);
+        return res;
+    }
```

Comment:
Note: we can let it as a failsafe but I am not sure this will happen. Taylor exponential is used only in calculate_blob_gas_price. The max it can get in input is MAX_BLOB_GAS_PER_BLOCK. Strategies should reflect that even with MAX_BLOB_GAS_PER_BLOCK*2 this will be greater than 2***128.

The following test is passing
```python
    @given(
        factor=st.just(MIN_BLOB_GASPRICE),
        numerator=st.just(MAX_BLOB_GAS_PER_BLOCK).map(Uint),
        denominator=st.just(BLOB_GASPRICE_UPDATE_FRACTION),
    )
    def test_taylor_exponential(
        self, cairo_run, factor: Uint, numerator: Uint, denominator: Uint
    ):
        assert taylor_exponential(factor, numerator, denominator) == cairo_run(
            "taylor_exponential", factor, numerator, denominator
        )
```

Reply:
> I am not sure this will ever happen with current constatns

This will definitely trigger in ef-tests in case inputs are not the current blob gas target